judging-algorithm/README.md

```diff
@@ -0,0 +1,43 @@
+Judging Algorithm Usage:
+
```

Comment:
There's kind of a lot of files here

How do you feel about adding a csv folder and a json folder?
<img width="425" alt="Screenshot 2023-02-08 at 8 46 39 PM" src="https://user-images.githubusercontent.com/59738880/217695305-2d35c587-4162-40d1-aca4-e679e9c5428a.png">

Reply:
you could also format it to have data and output folders